.build/generateLangium.ts

```diff
@@ -0,0 +1,5 @@
+import { execSync } from 'child_process';
+
+export function generateLangium() {
+  execSync(`pnpm --prefix ${process.cwd()}/packages/parser exec langium generate`);
```

Comment:
> Ideally we should use langium's JavaScript API, instead of going through the CLI.

I asked langium team to [export `langium-cli` internal](https://github.com/eclipse-langium/langium/issues/1150#issue-1850357159) and they have in https://github.com/eclipse-langium/langium/commit/24a9ebaf442b22a0cfaba2a0fccad2d94c65a2b7, but it appears that it has [an issue](https://github.com/eclipse-langium/langium/issues/1150#issuecomment-1685121389) and we're currently waiting for this PR https://github.com/eclipse-langium/langium/pull/1160 to be merged.

Reply:
switch to `execFileSync` 2618b1ebf3bf9febcf2fb82abfd9edf2f66018f8.

Reply:
generate using `langium-cli` API 8d0ca2c87604fd8cdae97799d72c6f3dba9ecf5e